src/App.tsx

```diff
@@ -2,17 +2,9 @@ import React from 'react';
 import './App.scss';
 
 import todosFromServer from './api/todos';
-import usersFromServer from './api/users';
-
-import { User } from './types/User';
 import { Todo } from './types/Todo';
-
-function getUser(userId: number): User | null {
-  const foundUser = usersFromServer.find(user => user.id === userId);
-
-  // if there is no user with a given userId
-  return foundUser || null;
-}
+import { TodoList } from './components/TodoList';
+import { getUser } from './utils/userUtils';
```

Comment:
why do you need this helper
```suggestion
function getUser(userId: number): User | null {
  const foundUser = usersFromServer.find(user => user.id === userId);
  // if there is no user with a given userId
  return foundUser || null;
}
export const todos: Todo[] = todosFromServer.map(todo => ({
  ...todo,
  user: getUser(todo.userId),
}));
```

Reply:
@etojeDenys в предыдущем комментарии ментор @VitaliyBondarenko1982 порекомендовал мне записать функцию getUser в utils . Зачем мне снова добавлять эту функцию в App.tsx?